pkg/views/prune/switch_profile.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package prune
+
+import (
+	"log"
+
+	"github.com/charmbracelet/huh"
+	"github.com/daytonaio/daytona/pkg/views"
+)
+
+func SwitchProfilePrompt(profileSwitchCheck *bool) {
```

Comment:
I would rename this to `SwitchToDefaultPrompt` and move it to the profiles view package. There's no reason this should only be scoped to the prune command.